.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    working_directory: ~/ezcater
+    docker:
+      - image: circleci/ruby:2.4
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+            - ruby-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
+            - ruby-cache-{{ .Branch }}-
+            - ruby-cache-
+
+      # Bundle install dependencies
+      - run: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4 --retry=3
+      - run: bundle clean --force
+
+      # Store bundle cache
+      - save_cache:
+          key: ruby-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
+          paths:
+            - vendor/bundle
+
+      # Run Rubocop
+      - run:
+          name: RuboCop
+          command: ruby bin/circle_rubocop.rb
```

Comment:
I think this is probably overkill and we should just always run `bundle exec rubocop` for the gem. 

Hopefully none of our gems get large enough to require this!

Reply:
Why rely on us doing that locally when we can guarantee on the server? Seems like a "best practice" we should be encouraging to me. Thoughts?

Reply:
I'm not saying to rely on doing that locally. I'm just saying run `bundle exec rubocop` here instead of the `cirle_rubocop.rb` script.

Reply:
👍